src/USDA.NET/Food/Search/SearchOptions.cs

```diff
@@ -0,0 +1,9 @@
+namespace USDA.NET.Food.Search
+{
+	public class SearchOptions
+	{
+		public string SearchTerm { get; set;}
```

Comment:
The lack of `JsonProperty` here is inconsistent.

Reply:
this class is flattened into the querystring see FoodCompositionClient.SearchAsync() so these do not need the JsonProperty